crates/driver/openapi.yml

```
@@ -279,94 +337,110 @@ components:
             Information about tokens used in the auction.
         deadline:
           $ref: "#/components/schemas/DateTime"
+        scoreCap:
+          description: |
+            The protocol rewards the winning solver the difference between the observed score and the second best score up to this cap.
+          $ref: "#/components/schemas/BigUint"
     SolveResponse:
       description: |
         Response of the solve endpoint.
+      type: array
+      items:
+        type: object
+        properties:
+          solutionId:
+            description: |
+              The unique identifier of the solution.
 
-        A score of 0 indicates the empty solution, which is never executed.
-      type: object
-      properties:
-        score:
-          description: |
-            The objective value of the solution.
-          type: number
-        signature:
-          description: |
-            Signature confirming that the Solver promised to have this solution for this auction.
+              This id is used to identify the solution when executing it.
+            type: integer
+          score:
+            description: |
+              The objective value of the solution.
+            type: number
+          submissionAddress:
+            description: |
+              The address that should be used to submit the solution.
```

Let's make it more clear that the protocol expects the information to be correct.
```suggestion
              The address that will be used to submit the solution.
```